File: app/src/main/java/com/tebet/mojual/view/bankconfirm/BankConfirmFragmentProvider.java
package com.tebet.mojual.view.bankconfirm;

import com.tebet.mojual.view.qualitydetail.OrderDetailFragmentModule;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class BankConfirmFragmentProvider {

    @ContributesAndroidInjector(modules = OrderDetailFragmentModule.class)
    abstract BankConfirmFragment provideQualityFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/bankconfirm/BankConfirmFragmentProvider.kt
package com.tebet.mojual.view.bankconfirm

import com.tebet.mojual.view.qualitydetail.OrderDetailFragmentModule

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class BankConfirmFragmentProvider {

    @ContributesAndroidInjector(modules = [OrderDetailFragmentModule::class])
    internal abstract fun provideQualityFragmentFactory(): BankConfirmFragment
}
